src/features/content-answers/ContentAnswers.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+
+import ContentAnswersOpenButton from './ContentAnswersOpenButton';
+
+const ContentAnswers = () => {
+    const [isModalOpen, setIsModalOpen] = useState(false);
+
+    const handleClick = () => {
+        setIsModalOpen(!isModalOpen);
```

Comment:
writing like this for now to prevent lint issue that `isModalOpen` variable is declared but not used.

Reply:
could we add a note to the modal ticket to revert this to use the `prevState`? otherwise it's possible that this toggles the incorrect value

Reply:
not sure I follow this conversation, could one of you explain?

Reply:
This should be written as `setIsModalOpen(prevState => !prevState)`, but it will leave `isModalOpen` not used which would cause error for lint testing.